src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -337,6 +337,46 @@ internal Authority GetAuthority(IAccount account)
             return await handler.RunAsync(CancellationToken.None).ConfigureAwait(false);
         }
 
+        internal async Task<AuthenticationResult> AcquireByRefreshTokenCommonAsync(IEnumerable<string> scopes, string userProvidedRefreshToken)
+        {
+            var context = CreateRequestContext();
+            SortedSet<string> _scopes;
+
+            if (scopes == null || scopes.Count() == 0)
+            {
+                _scopes = new SortedSet<string>();
+                _scopes.Add(ClientId + "/.default");
+                context.Logger.Info("No scopes provided for acquire token by refresh token request. Using default scope instead.");
```

Comment:
>"No scopes provided for acquire token by refresh token request. Using default scope instead." [](start = 36, length = 93)

for another PR... but these are also string literals and should ideally be in their own file.

Reply:
done